proposals/lip-0044.md

```diff
@@ -99,10 +123,10 @@ validatorAccountSchema = {
 
 ##### Properties and Default values
 
-The validator account holds the generator and BLS keys of a registered validator. In this section, we describe the properties of a validator account. These properties are set by the `registerValidatorKeys` function, called for instance during the processing of the [delegate registration command][lip-0057] or the [authority registration command][lip-0047].
+The validator account holds the generator and BLS keys of a registered validator. In this section, we describe the properties of a validator account. These properties are set by the [`registerValidatorKeys` function](#registerValidatorKeys), called for instance during the processing of the [delegate registration command][lip-0057] or the [authority registration command][lip-0047].
 
-* `generatorKey`: The public key whose corresponding private key is used to sign blocks generated by the validator. A valid generator key is 32 bytes long.
-* `blsKey`: The validator BLS key is the public BLS key whose corresponding private key is used to sign certificates. A valid BLS key is 48 bytes long.
+* `generatorKey: ED25519_PUBLIC_KEY_LENGTH`: The public key whose corresponding private key is used to sign blocks generated by the validator.
+* `blsKey: BLSKey`: The validator BLS key is the public BLS key whose corresponding private key is used to sign certificates.
```

Comment:
So far we only use type hinting for Python code mainly in order to skip validations for input parameters (e.g., byte length for IDs) and also to improve readability. I am not sure we should do this for schemas as well because all validations should already be part of the schema and the one sentence explanation should already clarify the type of data stored there.

Reply:
ok, I'll remove the hinting here